src/modality-specific-files/electromyography.md

```diff
@@ -0,0 +1,650 @@
+this extension when referring to it in the context of the academic literature.
+
+!!! example "Example datasets"
+
+    Electromyography datasets formatted according to this specification are available on
+    the [BIDS examples repository](https://github.com/bids-standard/bids-examples#emg)
+    and can be emulated when curating new datasets.
+
+## EMG data
+
+{{ MACROS___make_filename_template(
+"raw",
+datatypes=["emg"],
+suffixes=["emg", "events"])
+}}
+
+EMG device manufacturers use a variety of formats for storing raw data, and there is
+no single standard that all researchers agree on. For BIDS, EMG data MUST be
+stored in one of the following formats:
+
+| **Format**                                                         | **Extension(s)**         | **Description**                                                                                                                                                                                      |
+| ------------------------------------------------------------------ | ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Biosemi data format](https://www.biosemi.com/faq/file_format.htm) | `.bdf`                   | Each recording consists of a single `.bdf` file. [`bdf+`](https://www.teuniz.net/edfbrowser/bdfplus%20format%20description.html) files are permitted. The capital `.BDF` extension MUST NOT be used. |
+| [European data format](https://www.edfplus.info/)                  | `.edf`                   | Each recording consists of a single `.edf` file. [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted. The capital `.EDF` extension MUST NOT be used.                           |
+
+It is RECOMMENDED to use the BDF+ data format.
+Future versions of BIDS may extend this list of supported file formats.
+File formats for future consideration MUST have open access documentation, MUST have
+open source implementation for both reading and writing in at least two programming
+languages and SHOULD be widely supported in multiple software packages.
+Other formats that may be considered in the future should have a clear added advantage
+over the existing formats and should have wide adoption in the BIDS community.
+
+The [`acq-<label>`](../appendices/entities.md#acq) entity MAY be used to indicate
+simultaneous acquisition of data from multiple EMG devices, in cases where the devices
+store data in separate data files.
+If separate devices are being used with separate sampling rates, start times, or other
+acquisition parameters, the `acq-<label>` entity MUST be used to distinguish between them.
+The synchronization of data from multiple devices SHOULD be described in the
+[Scans](../modality-agnostic-files.md#scans-file) (`scans.tsv`) file, using the
+`acq_time` entity.
```

Comment:
```suggestion
`acq_time` column.
If the separate recordings include a channel that records synchronization events or a
per-frame latency signal, it is RECOMMENDED to name this channel as `LATENCY`.
```

Reply:
Yes, the `LATENCY` would provide the time for a sample since recording onset

Reply:
I'm going to resolve this suggestion, as it's incorrect regardless of whether we recommend `SYSCLOCK` or `LATENCY` as the preferred channel name. It's incorrect because such channel names are inappropriate for channels containing synchronization events (not timestamps); those channels should be called `TRIG` I guess)